pkg/sql/sem/tree/datum.go

```diff
@@ -2701,6 +2806,38 @@ func (t *DTable) ResolvedType() types.T {
 	return t.ValueGenerator.ResolvedType()
 }
 
+// AsJSON implements transformation to JSON
+func (t *DTable) AsJSON() (json.JSON, error) {
```

Comment:
This doesn't seem to be tested, and I can't seem to find the corresponding docs on how this works in PG? Can you point me to it (or show me an example?)